### ISSUE TITLE
On-board led (pin 13) not working on Galileo board

### ISSUE THREAD

**Issue by nicopernas** [emutex/wiring-x86#1]
On Arduino compatible boards the on-board general purpose led (GP) is connected to GPIO pin 13. On Galileo board the on-board pin is not currently working due to a known issue on the board itself. On Galileo GP led is not really connected to pin 13 but 20. A workaround needs to be put in place to write to GPIO pin 20 every time pin 13 is written. Extra muxing might be needed as well.

**Comment by DrBomb** [comment]
I've just checked this on the code, and pin 13 is mapped to gpio39 as expected on a Intel Galileo Gen 1, but the onboard LED is connected to gpio3.
I suppose the Arduino IDE has a workaround which maps both gpio39 and gpio3 to the same value so when pin 13 comes up, both physical pin 13 and the LED have HIGH outputs.

**Comment by nicopernas** [comment]
Yes, the IDE has a workaround to solve the HW issue. Patches are very welcome if you feel like it :)
